setup.py

```diff
@@ -30,7 +30,7 @@
     classifiers=[],
     packages=find_packages(),
     package_data={'chainerio' : package_data},
-    extras_require={'test':['pytest', 'flake8']},
+    extras_require={'test':['pytest', 'flake8', 'autopep8']},
```

Comment:
Not now, but I'd rather feel like moving on to newer and more popular formatters like [black](https://github.com/python/black) .